websockets/constants.js

```diff
@@ -4,10 +4,31 @@ var PORT = "{{ports[ws][0]}}";
 var PORT_SSL = "{{ports[wss][0]}}";
 var PORT_H2 = "{{ports[h2][0]}}";
 
+function url_has_variant(variant) {
+  if (typeof URLSearchParams === "function") {
```

Comment:
On what platform(s) does this check fail?

Reply:
I removed that check.